Program.cs

```diff
@@ -91,6 +96,7 @@ public static void UpdatePerson()
             }
         }
 
+        // deleting wanted person from the book
         public static void RemovePerson()
 
         {
```

Comment:
When there is no person inside the list, the method still asks for ID from the user. Please exit from the method when it happens.
Also, it is good to see all persons before start removing a person to ease the operation.